tsdb/meta.go

```diff
@@ -35,17 +34,9 @@ func NewDatabaseIndex() *DatabaseIndex {
 	return &DatabaseIndex{
 		measurements: make(map[string]*Measurement),
 		series:       make(map[string]*Series),
-		names:        make([]string, 0),
 	}
 }
 
-// Names returns a sorted list of measurement names.
-func (d *DatabaseIndex) Names() []string {
```

Comment:
Why not leave this function in place, and walk the keys in the `measurements` map?